source/js/buyers-guide/components/creep-vote/creep-vote.scss

```diff
@@ -111,4 +74,12 @@ $btn-shadow-width: 3px;
   .help-text {
     font-style: italic;
   }
+
+  .creep-chart {
+    min-width: 256px;
+  }
```

Comment:
The design of the chart looks the same as what we have on prod. Wondering why we need to set `min-width` now and why `256px`?

Reply:
Meant to remove that, gone now.